packages/builder/src/internal/cli/project-creation.ts

```diff
@@ -9,9 +9,6 @@ import { getPackageJson, getPackageRoot } from "../util/package-info";
 
 import { emoji } from "./emoji";
 
-const CREATE_SAMPLE_PROJECT_ACTION = "Create a sample project";
-const CREATE_EMPTY_BUILDER_CONFIG_ACTION = "Create an empty builder.config.js";
-
 const SAMPLE_PROJECT_DEPENDENCIES = [
   "@nomiclabs/builder-waffle",
   "ethereum-waffle",
```

Comment:
`ethereum-waffle` and `builder-waffle` this should be removed

Reply:
These are printed on startup.